distro.py

```diff
@@ -36,6 +36,17 @@
 import subprocess
 import sys
 
+try:
```

Comment:
This won't work if a user copy-pastes the `distro.py` file into their project which I believe is something we support. Maybe this is another reason to have `distro.__version__` be the source of the package version?